MainPerson/src/com/beyond/person/core/service/interfaces/BasePersonAction.java

```diff
@@ -0,0 +1,23 @@
+package com.beyond.person.core.service.interfaces;
+
+import com.beyond.person.core.model.BasePerson;
+
+public interface BasePersonAction {
+    /**
+     * method walk same for all
+     * @param person
+     */
+    default void walk(BasePerson person){
```

Comment:
Why do you need default ?

Reply:
for it to work the same way for all the people, as i dont have abstract class basepersonImpl, thats why i used this